cmake/GtsamPybindWrap.cmake

```diff
@@ -69,6 +69,14 @@ function(pybind_wrap
                      APPEND)
 
   pybind11_add_module(${target} ${generated_cpp})
+
+  if(APPLE)
+    # Darker sides of the RTTI system :)
```

Comment:
Please actually explain the issue and the solution, with the links as backup, only. Resist jokes.

Reply:
Copy that :)

Reply:
Done.